react/week3/my-app/src/index.js

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import Counter from "./counter";
+import moment from 'moment'; 
+import 'react-datepicker/dist/react-datepicker.css';
+import Example from "./react-datepicker";
+
+
+const TodoListItems=(props)=>{
+  const editstate = props.todoitem.editstate;
+  const details = editstate ? 
+                  <input type="text" onChange={props.UpdateTodo} value={props.todoitem.description} /> : 
+                  <div><div>{props.todoitem.description}</div>  <div>{props.todoitem.deadline}</div></div>
+    return <li className={ (props.todoitem.completed) ? "done": "not-done" } onSubmit={props.Updatedesc}>  
+      <div> {details} </div>
+      <button onClick={props.EditTodo} >{editstate ? "Update" : "Edit"}</button>
+      <button onClick={props.DeleteTodo} >Delete</button>
+      <input type="checkbox" checked={props.todoitem.completed} onChange={props.ToggleCompleted}/>
+      <label></label> 
```

Comment:
I am not sure if an empty item has any effect. Does this show on the screen?

Reply:
Yes, This is for checkbox state:for checked and unchecked

Reply:
I see now. Makes sense.